tests/__init__.py

```diff
@@ -0,0 +1,20 @@
+import requests
+from pyinfoblox import InfobloxWAPI
+
+class BaseTestCase(object):
+
+    def __init__():
+        self.infoblox = InfobloxWAPI(
+                 username='admin'
+                 password='admin'
+                 wapi='https://localhost:8080/wapi/v1.1/',
+                 verify=False)
+        
+        with open('./test_data/host.json') as jsonfile:
```

Comment:
This won't work, since the file will be closed once done with it. The same issue applies to the other `with open ...` statements.

Reply:
Reading and assigning it to a class var, that's then inherited preserves the var scope, so this does work. 
https://stackoverflow.com/questions/45100271/scope-of-variable-within-with-statement

Reply:
I was referring to the fact that you don't read anything from the file within the context manager, so basically the `host_base` property now points to a closed handle.


```python

>>> class Foo:
...     def __init__(self):
...         with open('/etc/passwd') as f:
...             self.some_file = f
...
>>>>>> class Bar(Foo):
...     def __init__(self):
...         super().__init__()
...
>>>
>>> bar = Bar()
>>> bar.some_file
<_io.TextIOWrapper name='/etc/passwd' mode='r' encoding='UTF-8'>
>>>
>>> bar.some_file.readlines()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: I/O operation on closed file.
```

Reply:
Yeah that was dumb, forgot I hadn't pushed the json.loads(jsonfile)